package day44_Exceptions;

public class Checked_Exception {

    public static void main(String[] args) {

        System.out.println("Test started");

        // Thread.sleep(3000);

        System.out.println("Completed");
    }



}
